examples/0_basic/0_create_did.rs

```diff
@@ -21,10 +20,25 @@ use iota_client::secret::SecretManager;
 use iota_client::Client;
 
 /// Demonstrates how to create a DID Document and publish it in a new Alias Output.
+///
+/// This example can be easily adapted to run in a private tangle by changing
+/// the network_endpoint and faucet_endpoint. In this example we use
+/// the Shimmer Testnet API.
+///
+/// A private tangle can be run by following the instructions in
+/// https://github.com/iotaledger/hornet/tree/develop/private_tangle.
 #[tokio::main]
 async fn main() -> anyhow::Result<()> {
+  // The API endpoint of a hornet node.
+  // In a locally running hornet node, this would usually be `http://127.0.0.1:14265`.
+  let network_endpoint: &str = "https://api.testnet.shimmer.network/";
+
+  // The faucet endpoint from where we can request funds for testing purposes.
+  // In a locally running hornet node, this would usually be `http://127.0.0.1:8091/api/enqueue`.
+  let faucet_endpoint: &str = "https://faucet.testnet.shimmer.network/api/enqueue";
```

Comment:
Capitalise "Hornet". Alternatively "HORNET" is used officially I think.

```suggestion
  // For a locally running Hornet node, this would usually be `http://127.0.0.1:14265`.
  let network_endpoint: &str = "https://api.testnet.shimmer.network/";

  // The faucet endpoint allows requesting funds for testing purposes.
  // For a locally running Hornet node, this would usually be `http://127.0.0.1:8091/api/enqueue`.
  let faucet_endpoint: &str = "https://faucet.testnet.shimmer.network/api/enqueue";
```